admin/client/ColorSchemeDropdown.tsx

```diff
@@ -0,0 +1,129 @@
+import React = require("react")
```

Comment:
A very small thing here:

```suggestion
import * as React from "react"
```
I _think_ this is identical? It's how it's imported elsewhere, do let me know if there is a difference though!

Reply:
You're right, I don't know how this ended up there but I think I just auto-imported it in VS Code. Well then, I'll change it tomorrow.